### ISSUE TITLE
MueLu test fails to build when Tpetra is disabled

### ISSUE THREAD

**Issue by mhoemmen** [trilinos/Trilinos#2339]
MueLu only optionally depends on Tpetra, but it fails to build when Tpetra (actually Kokkos, but Tpetra requires Kokkos) is disabled.  Here is the build error I get:
```
[ 98%] Building CXX object packages/muelu/test/unit_tests/CMakeFiles/MueLu_UnitTests.dir/Galeri.cpp.o
.../Trilinos/packages/muelu/test/unit_tests/Galeri.cpp:59:40: fatal error: TpetraCore_ETIHelperMacros.h: No such file or directory
 #include <TpetraCore_ETIHelperMacros.h>
                                        ^
compilation terminated.
```

@trilinos/muelu 

## Related Issues
* Blocks #2273 (downstream testing)

**Comment by mhoemmen** [comment]
Fixed in develop by PR #2314.

**Comment by jhux2** [comment]
@mhoemmen Thanks for fixing this.
